adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -48,6 +48,14 @@ var AdyenCheckoutHybris = (function () {
                 }
             }
 
+            if ( paymentMethod === "pos" ) {
+                var terminalId = document.getElementById('p_method_adyen_hpp_pos_terminal').value;
+                if( terminalId === "" ) {
+                    window.alert("Please select a terminal");
```

Comment:
can we make this text translatable ?

Reply:
We don't have internationalisation in java script alert messages. If we do it for this text, rest all text will also need to be changed to make all in one language.

Reply:
ok let's keep it as it is for now.